Index: Smarty/templates/Header.tpl
===================================================================
--- Smarty/templates/Header.tpl	(revisión: 191)
+++ Smarty/templates/Header.tpl	(copia de trabajo)
@@ -45,7 +45,7 @@
 	<!-- END -->
 	<script language="JavaScript" type="text/javascript" src="include/js/{php} echo $_SESSION['authenticated_user_language'];{/php}.lang.js?{php} echo $_SESSION['vtiger_version'];{/php}"></script>
 	<script language="JavaScript" type="text/javascript" src="include/js/QuickCreate.js"></script>
-	<script language="javascript" type="text/javascript" src="include/scriptaculous/prototype.js"></script>
+	{if $MODULE_NAME neq 'evvtApps'}<script language="javascript" type="text/javascript" src="include/scriptaculous/prototype.js"></script>{/if}
 	<script language="JavaScript" type="text/javascript" src="include/js/menu.js"></script>
 	<script language="JavaScript" type="text/javascript" src="include/calculator/calc.js"></script>
 	<script language="JavaScript" type="text/javascript" src="modules/Calendar/script.js"></script>
